docs/docs/04-Configurations/01-basic-configuration.md

```diff
@@ -85,6 +85,52 @@ const config = {
 When making this setting, make sure that the file names (including the path part) of different images do not cover each other.  
 Specifically, include the name, width, quality, and extension in the return value. If path is not included, all src's should be specified with `import` or `require` so that they can be distinguished by their hash value even if they have the same filename.
 
+### `sourceImageParser`
+
+- Type: function
+- Argument: Object
+- Return value: string
```

Comment:
Is it `ParsedImageInfo`?
Also, please describe what the value of `ParsedImageInfo` is!